src/desktop/apps/article/components/__tests__/Article.test.js

```diff
@@ -180,3 +180,96 @@ describe('<Article />', () => {
     rendered.state().following.length.should.exist
   })
 })
+
+describe('<Article /> without infinite scroll', () => {
```

Comment:
I added to two tests here to check that infinite scroll doesn't get initialize in the experiment group and that the `DisplayPanel` and `RelatedArticlesForCanvas` get rendered. Let me know if I'm missing anything else.